src/features/Progress/style.ts

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+
+interface ProgressBoxProps {
+  $maxWidth?: string;
+  $height?: string;
+  $boxBgColor?: string;
+}
```

Comment:
@rhehfl @bluetree7878 
```suggestion
type ProgressBoxProps = Pick<ProgressBarProps, '$maxWidth', | '$height' | '$boxBgColor'>
```
Pick 을 사용하면 ProgressBarProps를 재사용해서 인터페이스를 정의해볼 수 있을 것 같습니다 ~! 아래 InnerProgressProps도 마찬가지입니다. 

참고로 지금은 굳이 Pick과 같은 유틸리티 타입을 사용하지 않아도 될 것 같지만, 기본적으로 타입, 인터페이스도 중복 작성하기 보다는 최대한 중복 코드를 줄이는게 좋습니다 :)